### ISSUE TITLE
training results

### ISSUE THREAD

**Issue by chichengfengxue** [erdongsanshi/Fuss-Free-structure#7]
Thank you for your excellent work. When I was training the Shanghai Science and Technology A dataset, I strictly followed the configuration in the code, but the training results were quite different from those reported in the paper. Below is my log. The best **MAE** was achieved at **Epoch 684**.

<img width="624" height="465" alt="Image" src="https://github.com/user-attachments/assets/5c2a731f-be1a-4882-a79a-4abbb400d281" />

<img width="1056" height="67" alt="Image" src="https://github.com/user-attachments/assets/39af43b2-6f06-4b07-9783-5147a3f2f6a3" />

Looking forward to your reply.

**Comment by erdongsanshi** [comment]
You can try.

And I think the environment has certain influencing factors. You can try more other hyperparameters. Generally, the results can be stabilized at 50-52.

<img width="1080" height="366" alt="Image" src="https://github.com/user-attachments/assets/52d75e02-30a3-4cf9-ad60-4cee76259f30" />

**Comment by chichengfengxue** [comment]
> You can try.
> 
> And I think the environment has certain influencing factors. You can try more other hyperparameters. Generally, the results can be stabilized at 50-52.
> 
> <img alt="Image" width="1080" height="366" src="https://private-user-images.githubusercontent.com/109352239/527608184-52d75e02-30a3-4cf9-ad60-4cee76259f30.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU5ODE2MjYsIm5iZiI6MTc2NTk4MTMyNiwicGF0aCI6Ii8xMDkzNTIyMzkvNTI3NjA4MTg0LTUyZDc1ZTAyLTMwYTMtNGNmOS1hZDYwLTRjZWU3NjI1OWYzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxN1QxNDIyMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MWRjZTdmMmRhZjU2NDliODBmZjBmOWU5MjQ5YjI2Y2I4Yjk3MGU2MTRmZGQyNjY1M2UyNGZkOWVkZWQ1ZWVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ogMBPyXbDQRGX4i8cvNIgl39eCh1EJGCZI77K1vUPPM">

Thank you for your reply. I have completely followed the default settings in your code. Could you please tell me which hyperparameters I might need to adjust? Or perhaps I might need to adjust the learning rate?

**Comment by chichengfengxue** [comment]
> > You can try.
> > And I think the environment has certain influencing factors. You can try more other hyperparameters. Generally, the results can be stabilized at 50-52.
> > <img alt="Image" width="1080" height="366" src="https://private-user-images.githubusercontent.com/109352239/527608184-52d75e02-30a3-4cf9-ad60-4cee76259f30.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU5ODE2MjYsIm5iZiI6MTc2NTk4MTMyNiwicGF0aCI6Ii8xMDkzNTIyMzkvNTI3NjA4MTg0LTUyZDc1ZTAyLTMwYTMtNGNmOS1hZDYwLTRjZWU3NjI1OWYzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxN1QxNDIyMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MWRjZTdmMmRhZjU2NDliODBmZjBmOWU5MjQ5YjI2Y2I4Yjk3MGU2MTRmZGQyNjY1M2UyNGZkOWVkZWQ1ZWVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ogMBPyXbDQRGX4i8cvNIgl39eCh1EJGCZI77K1vUPPM">
> 
> Thank you for your reply. I have completely followed the default settings in your code. Could you please tell me which hyperparameters I might need to adjust? Or perhaps I might need to adjust the learning rate?



> You can try.
> 
> And I think the environment has certain influencing factors. You can try more other hyperparameters. Generally, the results can be stabilized at 50-52.
> 
> <img alt="Image" width="1080" height="366" src="https://private-user-images.githubusercontent.com/109352239/527608184-52d75e02-30a3-4cf9-ad60-4cee76259f30.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjYzMDkxNTEsIm5iZiI6MTc2NjMwODg1MSwicGF0aCI6Ii8xMDkzNTIyMzkvNTI3NjA4MTg0LTUyZDc1ZTAyLTMwYTMtNGNmOS1hZDYwLTRjZWU3NjI1OWYzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIyMVQwOTIwNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNTVhNTRiZWUzNWI0NjhhY2VkYmVmYmMyYmQ5ZmQzZDIwOTJiMWMyOTMzYmMyM2MyZDViNjI3MTMxMWQ4ZjhjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._i7pr6GiLfgAcxAaA7YFTTeRy_OvHpyHX0Fxc85dN9Y">




However, the metrics you reported for the SHA dataset in your paper are **48.3**. Could it be that I misunderstood something?
<img width="1202" height="581" alt="Image" src="https://github.com/user-attachments/assets/7aa7d4d0-c437-4005-b658-7bc0e4309299" />

**Comment by erdongsanshi** [comment]
Thank you for your reproduction. This model is relatively sensitive to hyperparameters (such as learning rate, batch size and regularization term, etc.) during the training process and may need to be adjusted to some extent according to the specific environment.

I have provided the model weights trained in our experimental environment in the warehouse. This model can achieve the 48.3 performance metric reported in the paper, which also verifies the reproducibility and performance ceiling of the method.

The differences in various environment configurations (such as PyTorch/CUDA/cuDNN versions or random initialization) may lead to certain fluctuations in the final results, which is a relatively common situation in current deep learning experiments. But the result will tend to a certain level.

**Comment by erdongsanshi** [comment]
Try the regularization Settings
